lerna.json

```diff
@@ -2,7 +2,6 @@
   "version": "independent",
   "npmClient": "npm",
   "packages": [
-    "api",
```

Comment:
Looks like that made it into the PR accidentally when excluding the API from the release. :slightly_smiling_face:
I think that's whats making the CI fail as it's not generating `version.ts` for the API.